databricks_cli/workspace/cli.py

```diff
@@ -136,6 +136,8 @@ def delete_cli(workspace_path, recursive):
 
 
 def _export_dir_helper(source_path, target_path, overwrite):
+    # normalize path to local filesystem (e.g. for Windows)
+    target_path = os.path.normpath(target_path)
```

Comment:
Why is this necessary? Shouldn't the user put in the normalized windows path in input to export_dir_cli?